fixtures/stairville/fog-m-1000.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "M-Fog 1000",
+  "shortName": "M-1000",
+  "categories": ["Hazer"],
+  "meta": {
+    "authors": ["Brioche"],
+    "createDate": "2023-12-12",
+    "lastModifyDate": "2023-12-12"
+  },
+  "comment": "Consommation de liquides : 35 ml/min\nTemps de chauffe : environ 2 min\nSortie continue maximale : environ 7 s\nTemps de rechargement : environ 6 s\nCapacité du réservoir de liquide : 2 l\nPuissance du brouillard : environ 240 m³ / min",
```

Comment:
Keep or remove?

Reply:
Fog output 240 m³ / min can be put into the `FogOutput` capability. The rest of the comment can be removed.